EmpireLegends/Player.cpp

```diff
@@ -234,3 +384,77 @@ Resources* Player::getResources() const
 	return pResources;
 }
 
+int Player::getNumofcards() {
+	return NumCards;
+}
+
+void Player::setNumofcards() {
+	NumCards++;
+}
+
+vector<Good*> Player::getListGoods()
+{
+	return playerGoods;
+}
+
+void Player::addGoods(Good* addedGood)
+{
+	playerGoods.push_back(addedGood);
```

Comment:
Just replace this line with a call to the `applyGood` method with the Resources instance